packages/fuchsia_remote_debug_protocol/examples/list_vms_and_flutter_views.dart

```diff
@@ -31,7 +31,8 @@ Future<Null> main(List<String> args) async {
   final String interface = args.length > 1 ? args[1] : '';
   // Example ssh config path for the Fuchsia device after having made a local
   // build.
-  const String sshConfigPath = '../../out/release-x86-64/ssh-keys/ssh_config';
+  const String sshConfigPath =
+      '../../../fuchsia/out/x64rel/ssh-keys/ssh_config';
```

Comment:
Blech... magic path. But I guess it's an example, so it's ok?

Reply:
Yeah... I'm not too thrilled about the magic path either.  I suppose I can change it to be an option in here.